app/assets/stylesheets/user.css.scss

```diff
@@ -2,25 +2,27 @@
     padding-top: 150px;
 }
 
+.form-background {
+  display: block;
+  background-color: #eeeeee;
```

Comment:
Color literals like `#eeeeee` should only be used in variable declarations; they should be referred to via variable everywhere else.